src/ddmd/statement.d

```diff
@@ -107,6 +124,7 @@ TypeIdentifier getThrowable()
 
 
 /***********************************************************
+ * Specification: http://dlang.org/spec/statement.html
```

Comment:
Just a random idea: what about having the specification inline in Ddoc comments and render the web page based on the Ddoc comments.